### PR TITLE
Issue-485 - Remove date range props.

### PR DESCRIPTION
**Pull request by benbcai**
Remove date range props and all references to it because there is no need for a date range in phase 1.

Resolves https://github.com/cerner/terra-core/issues/485

Thanks for contributing to Terra. 
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: packages/terra-date-picker/src/DatePicker.jsx [hunk 1]
```diff
@@ -52,15 +40,9 @@ const propTypes = {
    * An ISO 8601 string representation of the initial default date to show in the date input. This prop name is derived from react-datepicker but is analogous to defaultValue for a form input field.
    */
   selectedDate: PropTypes.string,
-  /**
-   * An ISO 8601 string representation of the start date for a date range.
-   */
-  startDate: PropTypes.string,
 };
 
 const defaultProps = {
```
Comment (benbcai) [inline packages/terra-date-picker/src/DatePicker.jsx]: Defaults for other props are added in this PR (https://github.com/cerner/terra-core/pull/452/files#diff-06f07d3cca7012107347dc63be00c147R65)

Comment (emilyrohrbough) [inline packages/terra-date-picker/src/DatePicker.jsx]: Looks like `endDate` and `startDate` defaults were added in the [PR](https://github.com/cerner/terra-core/pull/452/files#diff-06f07d3cca7012107347dc63be00c147R65) you listed.

**Comment by emilyrohrbough** [comment]
Do we remove the `DateRange.jsx` file as well?

**Comment by benbcai** [comment]
What the group agrees in the war room was to not expose any date range props/examples to consumers but still leave DateRange.jsx so that we have it we come back to use it again. It is there, just won't be used for now.

File: packages/terra-date-picker/src/DatePicker.jsx (hunk 1 above)
Comment (benbcai) [inline packages/terra-date-picker/src/DatePicker.jsx]: Yeah, there will be merge conflict when these are ready to be merged. I'll make sure that any reference to date range in that PR doesn't get merged down.

**Pull request merged**